spec/support/test_helpers/find_api_helper.rb

```diff
@@ -462,6 +462,137 @@ def stub_find_api_course(provider_code, course_code)
       "/courses/#{course_code}")
   end
 
+  def stub_find_api_provider_200_with_qualifications_and_program_type(
```

Comment:
Not sure if this needs to be in `spec/support/test_helpers/find_api_helper.rb`. Can it live in the only test file that currently uses it, `spec/services/sync_provider_from_find_spec.rb`?

Edit: I see there's a few more such methods in this file, so you're following an established pattern. In that case, do it this way, and if it gets unwieldy we can revisit and refactor. 👍